### PR TITLE
Blog posts

### PR DESCRIPTION
**Pull request by rogue707Dev**
Pull request to final building&fixing features

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -874,7 +874,76 @@ <h5> Features </h5>
         </div>
       </div>
     </section>
+    <!-- section blog post -->
+    <section class="section section-blog-post bg-dark">
+      <div class="container">
+
```
Comment (happimaker) [inline src/index.html:880]: Unnecessary empty lines.

File: src/styles/styles.scss [hunk 2]
```diff
@@ -816,6 +816,57 @@ section {
       }
```
Comment (happimaker) [inline src/styles/styles.scss:830]: Unnecessary empty lines

File: src/styles/styles.scss [hunk 3]
```diff
@@ -816,6 +816,57 @@ section {
...
+              &.badge {
+                background-color: white;
+                color: #27ae60;
+                padding:2px;
+                border-radius: 3px;
+              }
+              margin-right: 15px;
+            }
+          }
+
```
Comment (happimaker) [inline src/styles/styles.scss:889]: delete this empty line

File: src/styles/styles.scss [hunk 4]
```diff
@@ -759,21 +759,31 @@ section {
...
+        &.selected {
+          flex: 4;
+
+          .ticket-header {
+            padding: 62px 40px 60px;
+          }
+
+        }
         
         margin: 10px 10px;
```
Comment (happimaker) [inline src/styles/styles.scss:785]: You should code like this
margin: 10px;

File: src/styles/styles.scss [hunk 5]
```diff
@@ -912,6 +928,28 @@ section {
...
+        padding: 10px 0px;
+      }
+
+      img {
+        opacity: 0.5;
+
+        @media (max-width: $smalldeskBreakpoint) {
+          padding:10px 0px;
+        }
+
```
Comment (happimaker) [inline src/styles/styles.scss:962]: delete this unnecessary line.

File: src/styles/styles.scss [hunk 6]
```diff
@@ -912,6 +928,28 @@ section {
...
+      @media (max-width: $smalldeskBreakpoint) {
+        flex-direction: column;
+        margin: auto;
+        padding: 10px 0px;
+      }
+
+      img {
+        opacity: 0.5;
+
+        @media (max-width: $smalldeskBreakpoint) {
```
Comment (ghost) [inline src/styles/styles.scss:959]: Better to use smallDeskBreakpoint instead of smalldeskBreakpoint

**Pull request merged**